tests/unit-core/test-typedarray.c

```diff
@@ -303,6 +303,107 @@ test_typedarray_complex_creation (test_entry_t test_entries[], /**< test cases *
   }
 } /* test_typedarray_complex_creation */
 
+/**
+ * Test get/set/delete property by index.
+ */
+static void test_property_by_index (test_entry_t test_entries[])
+{
+  int test_int_numbers[5] = {-5, -70, 13, 0, 56};
+  double test_double_numbers[5] = {-83.153, -35.15, 0, 13.1, 89.8975};
+  uint8_t test_uint_numbers[5] = {83, 15, 36, 0, 43};
+
+  for (uint32_t i = 0; test_entries[i].constructor_name != NULL; i++)
+  {
+    jerry_value_t test_numbers[5];
+    uint32_t test_numbers_length = sizeof (test_numbers) / sizeof (jerry_value_t);
+    jerry_value_t typedarray = jerry_create_typedarray (test_entries[i].typedarray_type, test_numbers_length);
+    jerry_typedarray_type_t type = jerry_get_typedarray_type (typedarray);
+
+    switch (type)
+    {
+      case JERRY_TYPEDARRAY_INT8:
+      case JERRY_TYPEDARRAY_INT16:
+      case JERRY_TYPEDARRAY_INT32:
+      for (uint8_t j = 0; j < test_numbers_length; j++)
+      {
+        test_numbers[j] = jerry_create_number (test_int_numbers[j]);
+      }
+      break;
+      case JERRY_TYPEDARRAY_FLOAT32:
+      case JERRY_TYPEDARRAY_FLOAT64:
+      for (uint8_t j = 0; j < test_numbers_length; j++)
+      {
+        test_numbers[j] = jerry_create_number (test_double_numbers[j]);
+      }
+      break;
+      default:
+      for (uint8_t j = 0; j < test_numbers_length; j++)
+      {
+        test_numbers[j] = jerry_create_number (test_uint_numbers[j]);
+      }
+      break;
```

Comment:
Question: can we do this in one step? Create a single jerry value and set/get it?